extensions/amp-sidebar/1.0/sidebar.jss.js

```diff
@@ -48,10 +49,12 @@ const backdropClass = {
   position: 'fixed !important',
   top: '0 !important',
   left: '0 !important',
-  width: '100vw !important',
+  width: '120vw !important',
   height: '100vh !important',
+  overflow: 'hidden scroll !important',
```

Comment:
You shouldn't actually need to provide overflowing content to trigger `overscrollBehavior`, as long as you specify some overflow value of not `hidden`, which you do by specifying `scroll` for the Y axis here.

Reply:
Thanks Caroline!  That's what I was expecting how the CSS should work as well, but then when I was testing it out manually, I found that unless I provided overflowing content, it did not work as expected.  I tried testing in Storybook and also on a testPage and found that in Storybook it seems to behave like you described but in an arbitrary test page, overflowing content was needed.  I decided it made more sense to err on the side of caution.  What do you think?

Here's a test page I was working with: https://jsbin.com/jewivufuco/edit?html,output

If you copy the code into your own test page and run it locally (not on JSBin) you'll see what I mean.  The `overscroll` and `overflow` styling is on the `.messages` class.

Reply:
Is overflowing content needed? Or is `overflow: scroll` sufficient? With `scroll` it's in theory already clear that scrolling is involved. Unlike with `overflow: auto` it could go either way.

Reply:
Right, I was thinking `overflow: scroll` would be enough and it seems like it would work in theory, but when testing out manually it unfortunately did not appear to work.  If I am missing something though, happy to be corrected:

Here's the test page I was trying out: 
https://jsbin.com/jewivufuco/edit?html,output
(need to copy the code into your own local file to get it to see the issue) (for some reason it works fine in JSbin and storybook 😂😂)

Reply:
🤯  Here's the output link I reproduced with of your sample: https://output.jsbin.com/pajalad/1
This is also true for the lightbox changes when you don't have overflowing content (must opt into beta to get #32145): https://output.jsbin.com/yuvorey

It's possible the iframe used to display both the jsbin and storybook demos interfered with expected scrolling behavior, though I'm surprised that this should make a difference to scroll containment behavior.

Reply:
Right?  Isn't this so odd?  Thanks for sharing the output from jsbin, I didn't know you could do that.

So conclusion: elements must have overflowing content to trigger the `overscroll-behavior`, but for some reason in jsbin / storybook (likely due to iframe) the overflowing content is not needed.

I think in this case better to be safe and include the overflowing invisible element.

Reply:
Per offline conversation with @caroqliu .  We have decided to file a chrome bug to clarify the correct behavior and verify if this is a bug or not.  Then move forward after we hear back.

Bug is filed here: https://bugs.chromium.org/p/chromium/issues/detail?id=1176733